src/composables/fungibleTokens.ts

```diff
@@ -106,11 +109,36 @@ export function useFungibleTokens() {
     protocolsInUse,
     getLastActiveProtocolAccount,
   } = useAccounts();
+  const { getCurrentCurrencyRate } = useCurrencies();
 
   function getAccountTokenBalances(address: AccountAddress): ITokenBalance[] {
     return tokenBalances.value.filter((token) => token.address === address) || [];
   }
 
+  const accountsTotalTokenBalance = computed(() => {
+    const result = accounts.value.reduce(
```

Comment:
You can return the result immediatelly:
```suggestion
  const accountsTotalTokenBalance = computed(
    () => accounts.value
      .reduce((total, account) => {
        ...
      })
      .toFixed(2),
  );
```